datasource.go

```diff
@@ -73,6 +73,9 @@ type JSONData struct {
 	TLSSkipVerify     bool `json:"tlsSkipVerify,omitempty"`
 	httpHeaderNames   []string
 
+	// Used by Github
+	GitHubURL string `json:"githubUrl,omitempty"`
```

Comment:
Nit: hub shouldn't be capitalized
```suggestion
	GithubURL string `json:"githubUrl,omitempty"`
```

Reply:
Nvm, I just checked their page title and it is capitalized there 🤦